classes/order/Order.php

```diff
@@ -1088,8 +1088,6 @@ public static function getCustomerNbOrders($id_customer)
      */
     public static function getOrderByCartId($id_cart)
     {
-        Tools::displayAsDeprecated('Use Order::getIdByCartId() instead');
```

Comment:
@Balestrino Sometimes Travis has a hiccup. Rebase your PR in order to start the tests again :)